src/pages/NotFoundPage/NotFoundPage.module.css

```
@@ -0,0 +1,39 @@
+  width: 100%;
+  min-height: 100vh;
+  position: relative;
+  background-image: url("../../assets/NotFoundPage.png");
+  background-size: cover;
+  background-position: center;
+}
+
+.body {
+  position: absolute;
+
+  top: 44vh;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)